packages/react-i18n/src/babel-plugin/shared.ts

```diff
@@ -0,0 +1,23 @@
+import path from 'path';
+
+export const TRANSLATION_DIRECTORY_NAME = 'translations';
+export const DEFAULT_FALLBACK_LOCALE = 'en';
+
+export function getLocaleIds({
+  translationFilePaths,
+  fallbackLocale,
+}: {
+  translationFilePaths: string[];
+  fallbackLocale?: string;
+}): string[] {
+  return translationFilePaths
+    .map(filePath => path.basename(filePath, path.extname(filePath)))
+    .filter(locale => (fallbackLocale ? locale !== fallbackLocale : true))
```

Comment:
Does it make sense for this to be called without a `fallbackLocale`?  I think we should just throw an exception if this happens.